File: conversions/django/creature.py
class Creature(AbstractElementModel):

    # Physiology
    appearance = models.TextField(blank=True, null=True)
    weight = models.PositiveIntegerField(blank=True, null=True)
    height = models.PositiveIntegerField(blank=True, null=True)
    species = models.ManyToManyField("Species", blank=True, related_name="creature_species")

    # Lifestyle
    behaviour = models.TextField(blank=True, null=True)
    demeanour = models.TextField(blank=True, null=True)
    traits = models.ManyToManyField("Trait", blank=True, related_name="creature_traits")
    abilities = models.ManyToManyField("Ability", blank=True, related_name="creature_abilities")
    languages = models.ManyToManyField("Language", blank=True, related_name="creature_languages")

    # World
    birth_date = models.PositiveIntegerField(blank=True, null=True)
    location = models.ForeignKey("Location", on_delete=models.SET_NULL, blank=True, null=True, related_name="creature_location")
    territory = models.ForeignKey("Territory", on_delete=models.SET_NULL, blank=True, null=True, related_name="creature_territory")

    # Games
    lore = models.TextField(blank=True, null=True)
    senses = models.TextField(blank=True, null=True)
    hit_points = models.PositiveIntegerField(blank=True, null=True)
    armor_class = models.PositiveIntegerField(blank=True, null=True)
    challenge_rating = models.PositiveIntegerField(blank=True, null=True)
    speed = models.PositiveIntegerField(blank=True, null=True)
    tt_str = models.PositiveIntegerField(blank=True, null=True, validators=[MaxValueValidator(20)])
    tt_int = models.PositiveIntegerField(blank=True, null=True, validators=[MaxValueValidator(20)])
    tt_con = models.PositiveIntegerField(blank=True, null=True, validators=[MaxValueValidator(20)])
    tt_dex = models.PositiveIntegerField(blank=True, null=True, validators=[MaxValueValidator(20)])
    tt_wis = models.PositiveIntegerField(blank=True, null=True, validators=[MaxValueValidator(20)])
    tt_cha = models.PositiveIntegerField(blank=True, null=True, validators=[MaxValueValidator(20)])
    actions = models.ManyToManyField("Ability", blank=True, related_name="creature_actions")
    reactions = models.ManyToManyField("Construct", blank=True, related_name="creature_reactions")
    alignment = models.TextField(blank=True, null=True)

    def __str__(self):
        return self.name

File: conversions/csharp_unity_json_properties/Creature.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using Utils;

[System.Serializable]
public class Creature : Element
{
    [JsonProperty("appearance"), TextAttribute("")]
    public string appearance;
    [JsonProperty("weight"), Integer(0)]
    public int weight;
    [JsonProperty("height"), Integer(0)]
    public int height;
    [JsonProperty("species"), ReferenceAttribute(typeof(Species), true)]
    public string species;
    [JsonProperty("behaviour"), TextAttribute("")]
    public string behaviour;
    [JsonProperty("demeanour"), TextAttribute("")]
    public string demeanour;
    [JsonProperty("traits"), ReferenceAttribute(typeof(Trait), true)]
    public string traits;
    [JsonProperty("abilities"), ReferenceAttribute(typeof(Ability), true)]
    public string abilities;
    [JsonProperty("languages"), ReferenceAttribute(typeof(Language), true)]
    public string languages;
    [JsonProperty("birth_date"), Integer(0)]
    public int birthDate;
    [JsonProperty("location"), ReferenceAttribute(typeof(Location))]
    public string location;
    [JsonProperty("territory"), ReferenceAttribute(typeof(Territory))]
    public string territory;
    [JsonProperty("lore"), TextAttribute("")]
    public string lore;
    [JsonProperty("senses"), TextAttribute("")]
    public string senses;
    [JsonProperty("hit_points"), Integer(0)]
    public int hitPoints;
    [JsonProperty("armor_class"), Integer(0)]
    public int armorClass;
    [JsonProperty("challenge_rating"), Integer(0)]
    public int challengeRating;
    [JsonProperty("speed"), Integer(0)]
    public int speed;
    [JsonProperty("tt_str"), Integer(20)]
    public int ttStr;
    [JsonProperty("tt_int"), Integer(20)]
    public int ttInt;
    [JsonProperty("tt_con"), Integer(20)]
    public int ttCon;
    [JsonProperty("tt_dex"), Integer(20)]
    public int ttDex;
    [JsonProperty("tt_wis"), Integer(20)]
    public int ttWis;
    [JsonProperty("tt_cha"), Integer(20)]
    public int ttCha;
    [JsonProperty("actions"), ReferenceAttribute(typeof(Ability), true)]
    public string actions;
    [JsonProperty("reactions"), ReferenceAttribute(typeof(Construct), true)]
    public string reactions;
    [JsonProperty("alignment"), TextAttribute("")]
    public string alignment;
}
